e2e/app/element/base-element.ts

```diff
@@ -324,21 +324,15 @@ export default class BaseElement {
    * Click on element then wait for page navigation to finish.
    */
   async clickAndWait(timeout = 2 * 60 * 1000): Promise<void> {
-    try {
-      const navigationPromise = this.page.waitForNavigation({
-        waitUntil: ['load', 'domcontentloaded', 'networkidle0'],
-        timeout
-      });
-      await this.click({ delay: 10 });
-      await navigationPromise;
-    } catch (err) {
+    const navigationPromise = this.page.waitForNavigation({
+      waitUntil: ['domcontentloaded'],
+      timeout
+    });
```

Comment:
Reduce scope of `waitUntil` to be less restrict. Puppeteer waitForNavigation is flaky. It failed in CircleCI [run](https://app.circleci.com/pipelines/github/all-of-us/workbench/13012/workflows/d3a6a192-42c9-4b00-89ae-765c0b10244c/jobs/158600/tests) for `tests/workspace/workspace-edit.spec.ts`.
```
failure: [
  'TimeoutError: Navigation timeout of 120000 ms exceeded\n' +
    '    at /home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/LifecycleWatcher.js:106:111\n' +
    '    at FrameManager.waitForFrameNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:127:23)\n' +
    '    at Frame.waitForNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:441:16)\n' +
    '    at Page.waitForNavigation (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:794:16)'
]
```